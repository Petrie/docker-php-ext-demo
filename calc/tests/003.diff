003+ array(3) {
004+   [0]=>
005+   float(-17.222222222222)
006+   [1]=>
007+   float(-16.666666666667)
008+   [2]=>
009+   float(-16.111111111111)
010+ }