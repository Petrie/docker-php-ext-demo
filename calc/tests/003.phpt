--TEST--
Check for calc presence
--SKIPIF--
<?php if (!extension_loaded("calc")) print "skip"; ?>
--FILE--
<?php 
$mc = new mycalc();
echo $mc::add(3,4).PHP_EOL;
echo calc_sub(9,1).PHP_EOL;
var_dump(multiple_fahrenheit_to_celsius(
[1,2,3]
))
/*
	you can add regression tests for your extension here

  the output of your test code has to be equal to the
  text in the --EXPECT-- section below for the tests
  to pass, differences between the output and the
  expected text are interpreted as failure

	see php7/README.TESTING for further information on
  writing regression tests
*/
?>
--EXPECT--
7
8