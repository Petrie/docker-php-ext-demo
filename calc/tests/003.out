7
8
array(3) {
  [0]=>
  float(-17.222222222222)
  [1]=>
  float(-16.666666666667)
  [2]=>
  float(-16.111111111111)
}